src/solvers/dgsem_tree/indicators.jl

```diff
@@ -151,6 +151,41 @@ const IndicatorLoehner = IndicatorLöhner
 end
 
 
+"""
+    IndicatorIDP
+
+Indicator used for subcell shock-capturing used by Rueda-Ramírez, Pazner, Gassner.
+"""
+struct IndicatorIDP{Variable, Cache} <: AbstractIndicator
```

Comment:
I'm sure there will be other kinds of invariant domain preserving indicators. What about `IndicatorRPG`  (or `IndicatorRuedaRamirezPaznerGassner`?) to refer to this indicator? @gregorgassner, @sloede?